.github/workflows/platform-docker-publish-all-features-image.yml

```diff
@@ -120,7 +120,7 @@ jobs:
           context: .
           build-args: |
             SAML_INSTALLED=1
-            POETRY_OPTS=--with saml,auth-controller
+            POETRY_OPTS=--with saml,auth-controller,ldap
```

Comment:
Since we're resorting to a private git dependency, this step will fail without adding credentials to the git client.

See [one of the solutions](https://github.com/orgs/python-poetry/discussions/3794#discussioncomment-2469059).

Reply:
We are providing a distinct token to the checkout action for the private repos, aren't we?

Reply:
Yes, you're right, we are. Well caught, thanks!